cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -1019,13 +1098,13 @@ public ToolSearchResult searchTool(String keyword, String supportedPlatform, Str
         try {
             Request searchReq = getNewRequest(supportedPlatform, ballerinaVersion)
                     .get()
-                    .url(this.baseUrl + "/" + TOOLS + SEARCH_QUERY + keyword)
+                    .url(this.baseUrl + SEPARATOR + TOOLS + SEARCH_QUERY + keyword)
                     .build();
 
             logRequestInitVerbose(searchReq);
             Call httpRequestCall = client.newCall(searchReq);
             Response searchResponse = httpRequestCall.execute();
-            logRequestConnectVerbose(searchReq, "/" + TOOLS + "/" + SEARCH_QUERY + "/" + keyword);
+            logRequestConnectVerbose(searchReq, SEPARATOR + TOOLS + SEPARATOR + SEARCH_QUERY + SEPARATOR + keyword);
```

Comment:
Define a final variable to hold `SEPARATOR + TOOLS + SEPARATOR`?
e.g. Tool_PATH_PREFIX?

Reply:
Fixed with fc2c8abcedf